Formula/unit.rb

```diff
@@ -0,0 +1,95 @@
+class Unit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+  head "https://hg.nginx.org/unit", :using => :hg
+
+  option "with-debug", "Build with debug logging enabled"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  depends_on "go" => :optional
```

Comment:
Same here: please choose a “default” set of languages, that are likely to be used by most users, and ship those by default.

Reply:
This is trickier. Unit is designed to have shared objects for each runtime environment you want. A bit like php modules. And they are compiled with the same source tree as the daemon itself.

What if we had an nginx unit formula which came with shared objects linked against the base system's ruby, perl, python, then had an "nginx unit go" formula, an "nginx unit ruby" formula, etc, which link the nginx unit and go formulas, etc? This could be done without options, just more formulae.

Reply:
Not sure how much these would be used…